CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.7.14-alpha]
```

Comment:
This all looks great! The only thing we need to change is the version in the changelog:

```diff
- ## [0.7.14-alpha]
+ ## [Unreleased]
```

Our next released version may end up being 0.8.0 instead of 0.7.14. Thanks for the PR!

Reply:
That makes sense, changed it now.